go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/aws/aws-lambda-go
 go 1.12
 
 require (
-	github.com/stretchr/testify v1.4.0
+	github.com/aws/aws-sdk-go v1.31.12
```

Comment:
remove unused dependency

Reply:
Removed the unused dependency: e5194c9c68879d2971aa1d107b64025d09bf28df